opm/autodiff/FlowMainEbos.hpp

```diff
@@ -541,13 +568,51 @@ namespace Opm
         {
             bool output      = param_.getDefault("output", true);
             bool output_ecl  = param_.getDefault("output_ecl", true);
-            const Grid& grid = this->grid();
-            if( output && output_ecl && output_cout_)
+            if( output && output_ecl )
             {
-                const EclipseGrid& inputGrid = eclState().getInputGrid();
-                eclipse_writer_.reset(new EclipseWriter(eclState(), UgGridHelpers::createEclipseGrid( grid , inputGrid )));
-                eclipse_writer_->writeInitial(geoprops_->simProps(grid),
-                                              geoprops_->nonCartesianConnections());
+                const Grid& grid = this->globalGrid();
```

Comment:
Moved here manually as emails are not correctly threaded:

Yes, but that is not a problem. CpGrid is just a view wrapper of two CpGridData objects. It holds three pointer one to the global, one to the distributed, and on to the current one.